marlowe-playground-client/src/Language/Javascript/MarloweJS.ts

```diff
@@ -1,5 +1,7 @@
 import bignumber = require("bignumber.js")
```

Comment:
I think we use `big-integer.js` library everywhere, which uses native BigInt whenever possible.

Reply:
Kris just changed this, [now it is `bignumber`](https://github.com/input-output-hk/plutus/pull/2293/commits/6d56881132d8d584166bfa1e1dfe5442e429ec06#diff-6f6d69b59f3c9cc3160955dfe0a51c71L17). It seems it has the advantage that it serialises to JSON without using strings for numbers, which is quite neat.